common/eth2/src/types.rs

```diff
@@ -888,6 +888,27 @@ pub struct SseBlock {
     pub execution_optimistic: bool,
 }
 
+#[derive(PartialEq, Debug, Serialize, Deserialize, Clone)]
+pub struct SseBlobSidecar {
+    pub block_root: Hash256,
+    pub index: u64,
```

Comment:
I think we want to add quotes to the index here:
```suggestion
    #[serde(with = "serde_utils::quoted_u64")]
    pub index: u64,
```
the `Slot` type has this by default so we don't need it there

Reply:
Nice catch, thanks!